util.go

```diff
@@ -112,3 +112,7 @@ func revisionFromBuildInfo(info *debug.BuildInfo) string {
 
 	return ""
 }
+
+func Pointer[T any](v T) *T {
+	return &v
+}
```

Comment:
What's this used for?

Reply:
Seems I forgot to remove this. It was a helper function to create pointers.